cmd/incus-migrate/migrate_ova.go

```diff
@@ -311,11 +311,17 @@ func (m *OVAMigration) unpackOVA(outPath string) error {
 			continue
 		}
 
-		_, err = io.Copy(outFile, tarReader)
-		if err != nil {
-			fmt.Println("Error extracting file:", err)
-			outFile.Close()
-			continue
+		for {
+			_, err = io.CopyN(outFile, tarReader, 4*1024*1024)
+			if err != nil {
+				if errors.Is(err, io.EOF) {
+					break
+				}
+
+				fmt.Println("Error extracting file:", err)
+				outFile.Close()
+				continue
```

Comment:
This 'continue' will continue the inner, not outer loop.  But you've closed outFile, and would only re-open it if you continued the outer loop.

Reply:
Oops, yeah, we need to continue the outer loop, not the inner one... I'll send a fix.